### PR TITLE
Chore: sshkey client tests

### PR DESCRIPTION
**Pull request by razbensimon**
### Issue & Steps to Reproduce / Feature Request
[//]: <> (choose “resolves” “fixes” or “closes” and put link for the issue)
no unit tests for sshkey client

### Solution
add tests for sshkey client 
remove organization_id from public API. it is not necessary because of the organization cache.

### CODE REVIEW

**Comment by razbensimon** [comment]
[UT for client/sshkey.go](https://trello.com/c/gwmZwGdC/3356-ut-for-client-sshkeygo)

File: client/model.go [hunk 1]
```diff
@@ -149,7 +149,11 @@ type SshKey struct {
 }
 
 type SshKeyCreatePayload struct {
-	Name           string `json:"name"`
+	Name  string `json:"name"`
+	Value string `json:"value"`
+}
+
+type SshKeyCreatePayloadExtended struct {
+	SshKeyCreatePayload
```
Comment (roni-frantchi) [inline client/model.go:157]: Why the "extended"?  
Can we make it more similar to the other types?

Comment (razbensimon) [reply client/model.go:157]: I will be glad for a better name. 
We just need separated modes for Public API used by our users, and an Internal model for our internal REST api.

Comment (roni-frantchi) [reply client/model.go:157]: Why bother separating the two?  
How this entity different than say, Project?

Comment (razbensimon) [reply client/model.go:157]: We should do it for all the models contains organizationId. as discussed with @yaronya.
we want to expose fields that can be used by the user. it is weird to expose `organizationId` and do an unnecessary validation on our side - and throw an exception to the user. just do not expose it.

Comment (razbensimon) [reply client/model.go]: 45896c3

**Review by roni-frantchi — APPROVED** [review]

**Pull request merged**